api/profile/profileRouter.js

```diff
@@ -292,4 +292,44 @@ router.delete('/:id', (req, res) => {
   }
 });
 
+/**
+ * @swagger
+ * /profile/{id}/children:
+ *  get:
+ *    summary: To fetch children for each profile
+ *    description : Get the list of children for a particular profile id
+ *    security:
+ *      - okta: []
+ *    tags:
+ *      - profile
+ *    parameters:
+ *      - $ref: '#/components/parameters/profileId'
+ *    responses:
+ *      404:
+ *        $ref: '#/components/responses/NotFound'
+ *      200:
+ *        description: A profile object
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              properties:
+ *                message:
+ *                  type: string
+ *                  description: A message about the result
+ *                  example: Profile id found.
+ *                profile:
+ *                  $ref: '#/components/schemas/Profile'
+ */
+router.get('/:id/children', async (req, res) => {
+  const id = req.params.id;
+  Profiles.findChildren(id)
+    .then((children) => {
+      res.status(200).json(children);
+    })
```

Comment:
One other thing about this endpoint is that I am pretty sure that we are supposed to use crud ops, but ask Ash if it's okay if you don't.